src/main/resources/static/api.yml

```diff
@@ -48,12 +48,18 @@ paths:
           'text/csv;charset=UTF-8':
             schema:
               type: array
-              title: 'coursePrerequisites'
               items:
                 $ref: '#/components/schemas/PrerequisiteRecord'
       responses:
-        204:
+        200:
```

Comment:
Let's add some more context to the commit message as to why this is now a 200 response code.